#181414596 Fix Swagger Documentation

mucyoalexaxel opened the pull request:
#### What does this PR do?

1. It Fixes Requested Changes On PR [https://github.com/atlp-rwanda/cod-be/pull/20](#181414543 Configure BN Postgres Database & Sequelize ORM)
2. It Fixes Requested Changes On PR [https://github.com/atlp-rwanda/cod-be/pull/19](#181414596 Setup Swagger Documentation)

#### Description of Task to be completed?

1. Fixing All Changes Requested In the PRs Mentioned Above

#### How Should This Be Manually Tested?

1. Check The `Readme` file 

#### What are the relevant pivotal tracker stories?
Pivotal Tracker Story: [https://www.pivotaltracker.com/story/show/181414596](Setup API documentation using swagger)

SergeNm committed:
Setup Unit Test

SergeNm committed:
Setup Unit Test

SergeNm committed:
Setup Unit Test

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
# This is a combination of 28 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #3:

# This is a combination of 36 commits.
parent ba2c8cfd977c6f96299c49ca3da663b9016a88d3
author Alex Axel <mucyoalexaxel@gmail.com> 1646997011 +0200
committer Alex Axel <mucyoalexaxel@gmail.com> 1646997011 +0200

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #35:

fix .eslintrc

# This is the commit message #36:

fix .eslintrc

# This is the commit message #4:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

Setup Unit Test

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #11:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

Setup Unit Test

# This is the commit message #15:

fix .eslintrc

# This is the commit message #16:

fix .eslintrc

# This is the commit message #17:

fix .eslintrc

# This is the commit message #18:

fix .eslintrc

# This is the commit message #19:

fix .eslintrc

# This is the commit message #20:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #21:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #22:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #23:

Setup Unit Test

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #26:

# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

change .env.example

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 36 commits.
parent ba2c8cfd977c6f96299c49ca3da663b9016a88d3
author Alex Axel <mucyoalexaxel@gmail.com> 1646997011 +0200
committer Alex Axel <mucyoalexaxel@gmail.com> 1646997011 +0200

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #35:

fix .eslintrc

# This is the commit message #36:

fix .eslintrc

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

iyaremye-faustin reviewed (changes requested):
Seems to be a conflicting file please correct it and the rest will be ok

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
Merge branch 'develop' into ch-setup-API-Swagger-Documentation-#181414596

mucyoalexaxel committed:
# This is a combination of 30 commits.tree 287125b278f8da0b1bf29f48e50cc9e3e6fab07b
parent 0a5aad7212a7d64e2f164de3149685ecdcf00565
author Alex Axel <mucyoalexaxel@gmail.com> 1647095737 +0200
committer Alex Axel <mucyoalexaxel@gmail.com> 1647095737 +0200

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

fix .eslintrc

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

iyaremye-faustin reviewed (approved):
Requested change now made, It is ok and can be merged.
